lib/ronin/exploits/cli/commands/run.rb

```diff
@@ -382,6 +391,24 @@ def run_exploit
             end
           end
 
+          #
+          # Run the exploit's test method, and print the result.
+          #
+          def run_test
+            case (result = @exploit.test)
```

Comment:
Call `perform_test`. The `perform_*` methods wrap around the underlying `test`/`build`/`launch`/`cleanup` methods, and lets other `Mixins` modules override/hook without disrupting the exploit author's own `test`/`build`/`launch`/`cleanup` methods.